test/jdk/java/lang/ModuleTests/AnnotationsTest.java

```diff
@@ -146,23 +143,15 @@ public void testWithModuleInfoResourceXXXX() throws IOException {
      * Adds the Deprecated annotation to the given module-info class file.
      */
     static byte[] addDeprecated(byte[] bytes, boolean forRemoval, String since) {
-        ClassWriter cw = new ClassWriter(ClassWriter.COMPUTE_MAXS
-                                         + ClassWriter.COMPUTE_FRAMES);
-
-        ClassVisitor cv = new ClassVisitor(Opcodes.ASM6, cw) { };
-
-        ClassReader cr = new ClassReader(bytes);
-        List<Attribute> attrs = new ArrayList<>();
-        attrs.add(new ModuleTargetAttribute());
-        cr.accept(cv, attrs.toArray(new Attribute[0]), 0);
-
-        AnnotationVisitor annotationVisitor
-            = cv.visitAnnotation("Ljava/lang/Deprecated;", true);
-        annotationVisitor.visit("forRemoval", forRemoval);
-        annotationVisitor.visit("since", since);
-        annotationVisitor.visitEnd();
-
-        return cw.toByteArray();
+        return Classfile.parse(bytes).transform(ClassTransform.ACCEPT_ALL.andThen(ClassTransform.endHandler(clb -> {
```

Comment:
The conversion of this test okay but would be good if you split up the overly long lines as they are inconsistent with everything else in this test and makes it annoying to look at the changes side-by-side.